src/components/home/elements/SplitToken.vue

```diff
@@ -0,0 +1,666 @@
+      class="close btn-close"
+      @click="endSend">
+      <img src="@/assets/imgs/icons/operate/ic_close.svg">
+    </button>
+    <b-tabs @input="hideQrScan">
+      <b-tab title="Hot Wallet"
+             :disabled="!pageId || walletType==='coldWallet'"
+             :active="walletType==='hotWallet'">
+        <b-container
+          class="text-left"
+          v-if="pageId===1">
+          <b-form-group label="Maker Wallet Address"
+                        label-for="address-input">
+            <b-form-input id=address-input
+                          class="address-input"
+                          readonly
+                          v-model="address"
+                          :state="isValidMaker(address)"
+                          aria-describedby="inputLiveFeedback"></b-form-input>
+            <b-form-invalid-feedback id="inputLiveFeedback">
+              Cannot split token. You are not maker of this token.
+            </b-form-invalid-feedback>
+          </b-form-group>
+          <b-form-group>
+            <div class="unity">Current Unity {{ tokenUnity.toString() }}</div>
+          </b-form-group>
+          <b-form-group label="New Unity"
+                        label-for="amount-input">
+            <b-form-input id="recipient-input"
+                          class="recipient-input"
+                          type="text"
+                          v-model="newUnity"
+                          :state="isValidUnity()"
+                          aria-describedby="inputLiveFeedback">
+            </b-form-input>
+            <b-form-invalid-feedback id="inputLiveFeedback"
+                                     v-if="isNegative(this.newUnity)">
+              Negative number is not allowed.
+            </b-form-invalid-feedback>
+            <b-form-invalid-feedback id="inputLiveFeedback"
+                                     v-else-if="isBiggerThanMax()">
+              Too many significant digits for amount. Please reduce unity.
+            </b-form-invalid-feedback>
+            <b-form-invalid-feedback id="inputLiveFeedback"
+                                     v-else-if="!isNumFormatValid(this.newUnity)">
+              Invalid format.
+            </b-form-invalid-feedback>
+            <b-form-invalid-feedback id="inputLiveFeedback"
+                                     v-else-if="!isInteger()">
+              Integer number is allowed.
+            </b-form-invalid-feedback>
+            <b-form-invalid-feedback id="inputLiveFeedback"
+                                     v-else>
+              Invalid Unity.
+            </b-form-invalid-feedback>
+          </b-form-group>
+          <b-form-group>
+            <label class="fee-remark">Transaction Fee {{ formatter(fee) }} VSYS</label>
+            <span v-if="isInsufficient()"
+                  class="vsys-check">Insufficient VSYS balance</span>
+            <span v-if="!isSplit"
+                  class="vsys-check">This token cannot be split</span>
```

Comment:
Change the error message to `Cannot change unity. This token does not support split function.`